packages/p2p-v2/package.json

```diff
@@ -49,6 +49,10 @@
         "tailwindcss": "^3.3.6",
         "typescript": "^4.6.3",
         "webpack": "^5.81.0",
-        "webpack-cli": "^4.7.2"
+        "webpack-cli": "^4.7.2",
+        "tailwindcss": "^3.3.6",
+        "autoprefixer": "^10.4.16",
+        "postcss": "^8.4.24",
+        "history": "^5.0.0"
```

Comment:
`tailwindcss` and `postcss` are duplicated

Reply:
oof ill remove it asap